jenkins-integration/beaker/install/foss/30_install_dev_repos.rb

```diff
@@ -42,21 +42,17 @@ def get_latest_master_version(version)
 end
 
 def get_latest_agent_version
-  url = "https://jenkins-master-prod-1.delivery.puppetlabs.net/view/puppet-agent%20suite%20pipelines/job/platform_puppet-agent_intn-van-promote_suite-daily-promotion-master/lastSuccessfulBuild/api/json"
+  url = "http://builds.delivery.puppetlabs.net/passing-agent-SHAs/api/v1/json/report-master"
 
   uri = URI.parse(url)
   response = Net::HTTP.get_response(uri)
 
   if response.code == "200"
     json = JSON.parse(response.body)
-    actions = json["actions"].find { |hash| hash["_class"] == "hudson.model.ParametersAction" }
-    parameters = actions["parameters"]
-    pkg_build_param = parameters.find { |hash| hash["name"] == "SUITE_COMMIT" }
-
-    pkg_build_param["value"].strip
+    json["suite-commit"].strip
```

Comment:
Just to make sure, we're looking for the commit that corresponds to the latest puppet-agent build, yes?

Reply:
yes. it is used to fetch the repo configs and install the el7 agent package